src/caselawclient/models/documents/exceptions.py

```diff
@@ -2,5 +2,9 @@ class CannotPublishUnpublishableDocument(Exception):
     """A document which has failed publication safety checks in `Document.is_publishable` cannot be published."""
 
 
+class CannotEnrichUnenrichableDocument(Exception):
+    """We tried to enrich a document without a docx"""
```

Comment:
I don't think we should be describing the logic of the check in here; this should be something like:

```suggestion
    """A document which is not in an enrichable state as per `Document.can_enrich` cannot be enriched"""
```